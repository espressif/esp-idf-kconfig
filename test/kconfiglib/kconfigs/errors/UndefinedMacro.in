mainmenu "Testcase where nor macro nor environment variable is defined"


config MOTOR_NUMBER
    int "Number of motors"
    range 1 10
    default $(MAX_NUMBER_OF_MOTORS) # should cause error as nor macro nor envvar should be defined
    help
        Number of motors connected to the starship
