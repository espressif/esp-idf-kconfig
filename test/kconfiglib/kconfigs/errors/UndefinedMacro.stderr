macro expanded to blank string
